docs/output.md

```diff
@@ -58,19 +58,29 @@ Most plots are included in the HTML report (see above), but are also included in
 <summary>Output files</summary>
 
 - `tables/`
+
   - `annotation1/`: Directory containing annotation matrices generated in the course of analysis
     - `[array platform].annotation.tsv`: Annotations derived from an array platform
     - `[GTF name].anno.tsv`: Species wise annotations derived from a GTF in RNA-seq analysis
   - `processed_abundance/`: Directory containing processed abundance values from initial processing from e.g. DESeq2 or Affy:
-    - `[contrast_name].normalised_counts.tsv`: Normalised counts table (DESeq2)
+    - `[contrast_name].normalised_counts.tsv`: Normalized counts table (DESeq2). Rows are ensembl IDs and columns are samples.
```

Comment:
```suggestion
    - `[contrast_name].normalised_counts.tsv`: Normalized counts table (DESeq2). Rows are Ensembl IDs and columns are samples.
```

Uppercase seems to be correct (from a quick [web search](http://www.ensembl.org/Help/Faq?id=488)).